src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/config/Config.java

```diff
@@ -1127,6 +1175,25 @@ public void setRestVerified(boolean restVerified) {
         this.restVerified = restVerified;
     }
 
+    @Override
+    public int getEventTypesSize() {
```

Comment:
This function was added, but it doesn't look like it was used anywhere.

Reply:
It is not used in the java code. It's used in jelly to resize the list depending on the number of available types.

Reply:
Ditto for interface.